DigitalLearningSolutions.Web.Tests/TestHelpers/DiagnosticAssessmentHelper.cs

```diff
@@ -26,5 +26,26 @@ public static class DiagnosticAssessmentHelper
                 diagObjSelect
             );
         }
+
+        public static DiagnosticContent CreateDefaultDiagnosticContent(
```

Comment:
Don't you have this defined in another file as well? 🙂

Reply:
It's defined in a web test helper which I don't think we should be referencing from data tests?

Reply:
Actually that might not be the case

Reply:
Should be fine to reuse helpers across test projects anyway 🙂

Reply:
Is it okay to use data test helpers in the web tests? If so I can change this to use the data test helper

Reply:
See my comment above 🙂